index.js

```diff
@@ -228,7 +232,7 @@ module.exports = class BemEntityName {
      *
      * console.log(name); // BemEntityName { block: 'button' }
      *
-     * @param {integer} depth — tells inspect how many times to recurse while formatting the object.
+     * @param {number} depth — tells inspect how many times to recurse while formatting the object.
      * @param {object} options — An optional `options` object may be passed
      *                         	 that alters certain aspects of the formatted string.
```

Comment:
Here are huge offset, but [here](https://github.com/bem-sdk/bem-entity-name/pull/75/files#diff-168726dbe96b3ce427e7fedce31bb0bcR30) is smaller.